packages/contracts/deploy/update/to_v1.3.0/41_TokenVoting_Plugin.ts

```diff
@@ -16,10 +19,41 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deploy} = deployments;
   const [deployer] = await ethers.getSigners();
 
+  const zeroDaoAddress = ethers.constants.AddressZero;
+  const zeroTokenAddress = ethers.constants.AddressZero;
+  const emptyName = '';
+  const emptySymbol = '';
+  const emptyMintSettings: MintSettings = {
+    receivers: [],
+    amounts: [],
+  };
+
+  // Deploy the bases for the TokenVotingSetup
+  const governanceERC20DeployResult = await deploy('GovernanceERC20', {
+    contract: governanceERC20Artifact,
+    from: deployer.address,
+    args: [zeroDaoAddress, emptyName, emptySymbol, emptyMintSettings],
+    log: true,
+  });
+
+  const governanceWrappedERC20DeployResult = await deploy(
+    'GovernanceWrappedERC20',
+    {
+      contract: governanceWrappedERC20Artifact,
+      from: deployer.address,
+      args: [zeroTokenAddress, emptyName, emptySymbol],
+      log: true,
+    }
+  );
```

Comment:
We could re-use previously deployed `GovernanceERC20` or `GovernanceWrappedERC20` bases, as they didn't change (except the OZ version bump).

Should we do it? I would say it is better to pay a bit more for a fresh deployment (with the latest OZ updates) instead of having a potential address mix-up just to save a few bucks.

Other question: Will the new `active_contract.json` after the protocol upgrade contain the two governance token contract addresses automatically, or do we have to remember to add them there?
It could generally make sense to add all implementation/base addresses there.

Reply:
I would update them because they have changed with the OZ update.
Adding stuff to the active_contracts.json is a manual task and we can do that for sure